### ISSUE TITLE
Render Container - Only one class rendered

### ISSUE THREAD

**Issue by LukeTOBrien** [lunet-io/markdig#382]
Hello,

It seems that when I create a container with more than one class, only the first class is rendered.

The following Markdown:
```markdown
::: a-class another-class
:::
```
I would expect the following:
```html
<div class="a-class another-class"></div>
```

Instead I am getting only the first class rendered.  
I have created a [.Net Fiddle to demo](https://dotnetfiddle.net/hDyuZ3)

I guess this is an issue with HtmlRenderer.WriteAttributes ?

Thanks

**Comment by xoofx** [comment]
It is currently by design, not an issue with WriteAttributes, but it is similar to code fence where the only first element is taken and the following are extra arguments, but in the case of container they are probably not used. I don't remember the full details, but feel free to open a PR with appropriate tests if you want this behavior (and I don't remember if it could bring another issue)

**Comment by MihaZupan** [comment]
As Alexandre said only the first item (the Info) is added as a class. The rest are added to the Arguments property, which is ignored when rendering HTML (but is exposed on the AST).

You can add them to the attributes like so:
```c#
foreach (var container in document.Descendants<CustomContainer>())
{
    if (!string.IsNullOrEmpty(container.Arguments))
    {
        var attributes = container.GetAttributes();
        foreach (var argument in container.Arguments.Split(' '))
        {
            if (argument.Length != 0)
            {
                attributes.AddClass(argument);
            }
        }
    }
}
```

An option to render arguments as classes could ofc be exposed if you think it is commonly desired behavior.

**Comment by LukeTOBrien** [comment]
Okay understood, thanks for the code snippet.  
I could possibly have a go at adding this feature, it seems a shame to waste arguments.

**Comment by xoofx** [comment]
> I could possibly have a go at adding this feature, it seems a shame to waste arguments.

Yeah, the rationale of the name of a container is like a method name, and the arguments are on the side. You could develop a renderer that interprets them differently of generate a different HTML. The fact that the name of the container is put as a `class`  attribute is a default behavior for practical convenience. In case of adding arguments as class arguments as well is fine (for a default behavior), as long as we can switch it off by extension options.
